frontend/package.json

```diff
@@ -32,7 +32,9 @@
     "@web3-onboard/walletconnect": "^2.1.3",
     "bnc-notify": "^1.5.1",
     "core-js": "^3.6.5",
+    "ethereum-cryptography": "^1.1.2",
     "ethers": "^5.6.8",
+    "localforage": "^1.10.0",
```

Comment:
_(arbitrary comment location to start a thread)_

> I used aes encryption because it was more straightforward for me compared to xor, but am open to changing it if XOR makes sense.

Sounds good, let me check on this and get back to you

Reply:
Any updates here?